src/ontology/uberon-edit.obo

```diff
@@ -222488,6 +222488,16 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500108
+name: cheek teeth
+def: "The post-canine teeth (molars and premolars) that occur at the zygomatic plate." [Wikipedia:Cheek_teeth]
+synonym: "cheekteeth" EXACT []
+is_a: UBERON:0013164 ! molariform tooth
+relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+relationship: part_of UBERON:0013217 ! zygomatic plate
```

Comment:
the zygomatic plate is bony, is the cheek tooth really part of the bone?

compare with

```yaml
id: UBERON:0003267
name: tooth of upper jaw
def: "A calcareous tooth that is part of the upper jaw region. In mammals, the upper jaw teeth are attached to the maxilla." [http://orcid.org/0000-0002-6601-2165]
synonym: "maxillary tooth" NARROW [NCBITaxon:40674]
synonym: "upper jaw tooth" EXACT [MA:0001909]
xref: EHDAA:8045
xref: EMAPA:17938
xref: MA:0001909
xref: NCIT:C49792
xref: SCTID:422279005
xref: UMLS:C1710581 {source="ncithesaurus:Upper_Jaw_Tooth"}
intersection_of: UBERON:0001091 ! calcareous tooth
intersection_of: part_of UBERON:0001709 ! upper jaw region
relationship: attaches_to UBERON:0011597 ! bone of upper jaw
relationship: part_of UBERON:0009680 ! set of upper jaw teeth
property_value: editor_note "Note that we follow EHDAA2 and consider teeth part of the jaw skeleton" xsd:string
```

Reply:
I realized that I was looking at upper cheek teeth, but cheek teeth are both upper and lower, so I will change.